### DISCUSSION TITLE
Raspberry 5

### DISCUSSION THREAD

**Discussion by stephanlj** [brutella/hkknx-public#355] [Q&A] [answered]
How can I change from Raspi 3 to 5 and what do I have to do so that I do not receive an error message when I try to start HomeKit on the 5

**Comment by brutella** [comment, marked as answer]
Depending on how your configuration looks like, this is what you have to do.

- If you are using the "old" Raspberry Pi disk image, you cannot just simply put the sd card into the Raspberry Pi 5, because the installed operating system doesn't work for the Raspberry Pi 5. In this case you should
    - Create a backup of hkknx in Settings → More → Backup
    - Download the new Raspberry Pi disk image (currently the latest version if version 3.0) https://github.com/brutella/hkknx-public/releases/download/3.0.0/hkknx-3.0.0_rpi.zip
    - Install the disk image on an sd card
    - Put the sd card into the Rasbperry Pi 5 and open the web page http://hkknx.local:8080
    - Then import your backup
- If you are using the new Raspberry Pi disk image from version 3.0, you can simply insert the sd card into the Raspberry Pi 5.

**Comment by stephanlj** [comment]
Hi,vielen Dank für Deine schnelle Antwort, werde ich gleich versuchen.Gruß Stephan Von meinem iPad gesendet

**Comment by stephanlj** [comment]
Hi,
Everything is running, thanks again.
